keyboards/annepro2/annepro2.c

```diff
@@ -76,30 +73,34 @@ void OVERRIDE keyboard_post_init_kb(void) {
     wait_ms(15);
 
     // loop to clear out receive buffer from ble wakeup
-    while(!sdGetWouldBlock(&SD1))
-        sdGet(&SD1);
+    while (!sdGetWouldBlock(&SD1)) sdGet(&SD1);
+
+    annepro2LedGetStatus();
 
     keyboard_post_init_user();
 }
 
-void OVERRIDE matrix_init_kb(void) {
-    matrix_init_user();
-}
+void OVERRIDE matrix_init_kb(void) { matrix_init_user(); }
 
 void matrix_scan_kb() {
     // if there's stuff on the ble serial buffer
     // read it into the capslock struct
-    while(!sdGetWouldBlock(&SD1)) {
-        sdReadTimeout(&SD1, (uint8_t *) &BLECapsLock, sizeof(ble_capslock_t), 10);
+    while (!sdGetWouldBlock(&SD1)) {
+        sdReadTimeout(&SD1, (uint8_t *)&BLECapsLock, sizeof(ble_capslock_t), 10);
 
         // if it's capslock from ble, darken led
         if (BLECapsLock.caps_lock) {
-            annepro2LedClearMask(MATRIX_COLS * 2);
+            // annepro2LedClearMask(MATRIX_COLS * 2);
         } else {
-            annepro2LedSetMask(MATRIX_COLS * 2);
+            // annepro2LedSetMask(MATRIX_COLS * 2);
         }
     }
 
+    /* While there's data from LED keyboard sent - read it. */
+    while (!sdGetWouldBlock(&SD0)) {
+        uint8_t byte = sdGet(&SD0);
+        protoConsume(&proto, byte);
```

Comment:
This doesn't use protoSilence(); It could probably be added, but even without it the protocol will eventually synchronize (after some byte is dropped, etc.)